.github/workflows/runtime-harness.yml

```
@@ -104,31 +125,50 @@ jobs:
+            --url "http://127.0.0.1:${AI_SHIFU_WEB_PORT}/login" \
+            --timeout-seconds 120
+          python scripts/harness/runtime_timeline.py mark \
+            --output "$AI_SHIFU_RUNTIME_TIMELINE" \
+            --name fast-value-health \
+            --status finished
 
       - name: Resolve Playwright version
         id: playwright-version
         working-directory: src/cook-web
-        run: echo "version=$(node -p "require('./package-lock.json').packages['node_modules/playwright'].version")" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(node -p \"require('./package-lock.json').packages['node_modules/playwright'].version\")" >> "$GITHUB_OUTPUT"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:1:6: The mentioned syntax error was in this double quoted string

(shellcheck)